src/pages/clouddocs/release-notes.md

```diff
@@ -32,14 +40,19 @@ The following sections document the new features and changes that were included
     - Added support to allow matching CUI from the derived concept value.
     - Added support to propagate fields from the derived concept value first and use the secondary annotation as a fallback option.
     - Added support to promote dimension, frequency, and duration to the attribute if they exist in a derived concept value annotation.
-- Added warning messages when a processed ACD flow contains deprecated services.
+- Configuration Editor
+    - Added support to include linked attributes when exporting dictionary concepts.
+    - Integrated attribute search in the dictionary and UMLS concept views
+    - Context menus have been added to concept identifier and type tag fields to allow cut, paste, and delete of individual tags.
```

Comment:
for readability and to match the first two bullets, maybe:
- Added context menus to concept identifier and type tag fields to allow cut, paste, and delete of individual tags.
- Shows a warning message if a flow contains a deprecated service.